### ISSUE TITLE
Invalid files are also processed "successfully"

### ISSUE THREAD

**Issue by sushinoya** [nrgpy/nrgpy#10]
If a file does not exist, the output to the screen is still the same as when a valid file is processed by nrgpy_rld.local. Example - 
```python
nrgpy_rld.local(filename="file_that_doesnt_exist.rld", out_dir="some_dir")
```

Will print this to the console - 
```
file_that_doesnt_exist.rld ... 		[DONE]
Queue processed
```

Ideally, invalid paths or files should be reported differently to the user

**Comment by nrgpy** [comment]
good point, thank you for sharing that feedback! we'll put this one in the backlog to figure out. sorry for the confusion it may have caused.

**Comment by dave-c-vt** [comment]
fixed in nrgpy==1.5.0
